code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -1087,6 +1170,7 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 
 /mob/living/silicon/robot/update_icons()
 	overlays.Cut()
+	borg_icons()
```

Comment:
Why was it necessary to move this up here?

Reply:
No, turns out I moved it during debugging and just forgot to move it back. Has been changed back now.